recipes/cWB/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "cWB" %}
+{% set version = "6.4.2" %}
+
+package:
+   name: "{{ name|lower }}"
+   version: "{{ version }}"
+
+source:
+   url: https://gitlab.com/gwburst/public/library/-/archive/cWB-{{ version }}_test_conda_v2/library-cWB-{{ version }}_test_conda_v2.tar.gz
```

Comment:
This URL doesn't look like it should be used for real releases, is this a vestige of testing?

Reply:
Yes at the moment, since they haven't done a new release of cWB. This is just an intermediate naming to get cWB working in Conda and after this link will be moved to the official releases.

Reply:
This issue should be marked as resolved *after* the URL is moved to the official release.

Reply:
The url will be updated when there is a new release of `cWB`, but this will be independent of this package being put into Conda.

Reply:
We expect the conda package releases to match the releases of the upstream project, so we cannot release a "6.4.2" package that was not build from a "6.4.2" release. This avoids ambiguity about what was in the source code. In the future, if you want to make pre-releases for this package, we have a mechanism for that, but it is not available for the first release.


https://conda-forge.org/docs/orga/guidelines.html#intended-usage
https://conda-forge.org/docs/maintainer/knowledge_base.html#pre-release-builds

Reply:
The URL has been updated to the `6.4.2` official release.